velox/functions/sparksql/String.h

```diff
@@ -1431,4 +1431,37 @@ struct LevenshteinDistanceFunction {
   }
 };
 
+template <typename T>
+struct Empty2NullFunction {
```

Comment:
Perhaps add comments for this function following the style of https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/String.h#L771-L779.